sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/src/Listeners/ServiceBusScaleMonitor.cs

```diff
@@ -78,7 +78,7 @@ public async Task<ServiceBusTriggerMetrics> GetMetricsAsync()
                     var peekedMessages  = await _receiver.Value.PeekMessagesAsync(10, fromSequenceNumber: peekedMessage.SequenceNumber).ConfigureAwait(false);
                     foreach (var receivedMessage in peekedMessages )
                     {
-                        if (MessageIsActive(receivedMessage))
+                        if (MessageIsActive(receivedMessage) && !MessageIsLocked(receivedMessage))
```

Comment:
The message won't ever be locked when peeking. It may be locked by another consumer, but you are not able to determine that when peeking. The LockedUntil time will just be the default DateTimeOffset value.

Reply:
Ah right. Apologies for the invalid PR. I will close this and follow this up on the issue I created.